src/Compilers/Core/Portable/DocumentationCommentId.cs

```diff
@@ -344,21 +374,59 @@ public override void VisitMethod(IMethodSymbol symbol)
             public override void VisitNamespace(INamespaceSymbol symbol)
             {
                 _builder.Append("N:");
-                _generator.Visit(symbol);
+                AppendNamespace(symbol, _builder);
             }
 
             public override void VisitNamedType(INamedTypeSymbol symbol)
             {
                 _builder.Append("T:");
-                _generator.Visit(symbol);
+                AppendNamedType(symbol, _builder);
```

Comment:
>AppendNamedType(symbol, _builder); [](http://example.com/codeflow?start=16&length=34)

Similar comment as for the previous method, bath can be part of the same issue.

Reply:
wfm.